src/data/AkitaPaymentPointerData.js

```diff
@@ -30,6 +34,20 @@ class AkitaPaymentPointerData {
 		return newPaymentPointerData;
 	}
 
+	/**
+	 * It is expected that payment pointers are validated by Akita often. In order to make sure
+	 * that validation occurs often, we keep track of the last time the payment pointer was
+	 * validated. When Akita validates a payment pointer, the time it was validated can be
+	 * set using this function.
```

Comment:
The first sentence of this function description should be a concise summary of what it does. Any additional background can be provided afterwards.